### PR TITLE
Aws overusage

### PR DESCRIPTION
**Pull request by athiruma**
1. create policy cost_overusage
2. search Gmail, using LDAP

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -6,7 +6,7 @@ ARG VERSION
 # install gitleaks
 ARG gitleaks_version=7.0.2
 RUN apt-get update \
-     && apt-get install -y wget \
+     && apt-get install -y wget build-essential python3-dev libldap2-dev libsasl2-dev vim \
```
[the remark this answers is not in the document]
Comment (athiruma) [reply Dockerfile:9]: I tested in the container, and put this

File: cloud_governance/common/elasticsearch/elasticsearch_operations.py [hunk 2]
```diff
@@ -452,3 +452,15 @@ def get_elasticsearch_index_by_id(self, index: str, id: str):
         :return:
         """
         return self.__es.get(index=index, id=id)
+
+    @typechecked()
+    @logger_time_stamp
+    def get_index_hits(self, days: int, index: str):
+        search = Search(using=self.__es, index=index).filter('range', timestamp={'gte': f'now-{days}d', 'lt': 'now'})
+        search = search[0:10000]
```
[the remark this answers is not in the document]
Comment (athiruma) [reply cloud_governance/common/elasticsearch/elasticsearch_operations.py:460]: ok

File: cloud_governance/common/classes/elastic_upload.py [hunk 3]
```diff
@@ -0,0 +1,46 @@
+import os
```
[the remark this answers is not in the document]
Comment (athiruma) [reply cloud_governance/common/classes/elastic_upload.py:1]: It will be same for IBM & AWS

[the remark this answers is not in the document]
Comment (ebattat) [reply cloud_governance/common/classes/elastic_upload.py:1]: common/classes/elastic_upload
If it related to elastic let put it there 
I dont understand the goal of this class

**Pull request merged**